rules/sort-object-types-utils.ts

```diff
@@ -51,9 +51,7 @@ export let customGroupMatches = (props: CustomGroupMatchesProps): boolean => {
       props.elementName,
       props.customGroup.elementNamePattern,
     )
-    if (!matchesElementNamePattern) {
-      return false
-    }
+    return matchesElementNamePattern
```

Comment:
[maintainability] I personally prefer the old way: if we add another condition after this one later, we will need to change

```ts
return matchesElementNamePattern
```

to 

```ts
if (!matchesElementNamePattern) {
  return false
}
```

again so it doesn't exit early, and it's easy to forget.

Reply:
Back to the previous version.